modules/api/src/main/java/org/eclipse/xpanse/api/ServiceCatalogApi.java

```diff
@@ -157,28 +156,15 @@ private ServiceTemplateQueryModel getServiceTemplatesQueryModel(Category categor
         return query;
     }
 
-    private UserAvailableServiceVo convertToUserAvailableServiceVo(
-            ServiceTemplateEntity serviceEntity) {
-        if (Objects.nonNull(serviceEntity)) {
-            UserAvailableServiceVo userAvailableServiceVo = new UserAvailableServiceVo();
-            BeanUtils.copyProperties(serviceEntity, userAvailableServiceVo);
-            userAvailableServiceVo.setIcon(serviceEntity.getOcl().getIcon());
-            userAvailableServiceVo.setDescription(serviceEntity.getOcl().getDescription());
-            userAvailableServiceVo.setNamespace(serviceEntity.getOcl().getNamespace());
-            userAvailableServiceVo.setBilling(serviceEntity.getOcl().getBilling());
-            userAvailableServiceVo.setFlavors(serviceEntity.getOcl().getFlavors());
-            userAvailableServiceVo.setDeployment(serviceEntity.getOcl().getDeployment());
-            userAvailableServiceVo.setVariables(
-                    serviceEntity.getOcl().getDeployment().getVariables());
-            userAvailableServiceVo.setRegions(
-                    serviceEntity.getOcl().getCloudServiceProvider().getRegions());
-            userAvailableServiceVo.add(
+    private UserAvailableServiceVo convertToUserAvailableVo(
```

Comment:
convertToUserAvailableVo -> this method name is a bit misleading. Its better we call it `convertToUserAvailableServiceVo` as before.